src/MAT_HDF5.jl

```diff
@@ -290,12 +290,12 @@ elseif length(s) > 63
 end
 
 toarray(x::Array) = x
-toarray(x::Array{Bool}) = reinterpret(UInt8, x)
+toarray(x::Array{Bool}) = reinterpret.(UInt8, x)
```

Comment:
I think you can delete this
`toarray.(x)` should work if  `toarray(a)` works, where `a` is a Bool, be the definition below.

Reply:
It's necessary for correct matching during multiple dispatch.